src/operator/lrn.cc

```diff
@@ -38,9 +38,26 @@ Operator* LocalResponseNormProp::CreateOperatorEx(Context ctx, std::vector<TShap
 DMLC_REGISTER_PARAMETER(LRNParam);
 
 MXNET_REGISTER_OP_PROPERTY(LRN, LocalResponseNormProp)
-.add_argument("data", "NDArray-or-Symbol", "Input data to the ConvolutionOp.")
+.add_argument("data", "NDArray-or-Symbol", "Input data.")
 .add_arguments(LRNParam::__FIELDS__())
-.describe("Apply convolution to input then add a bias.");
+.describe(R"code(Applies Local Response Normalization to the input.
+
+Normalization helps in generalization and used to prevent neurons from 
```

Comment:
what I was aiming for was to write a sentence about Normalization and then describe LRN. I will remove it if this leads to confusion.
You are right that ReLU neurons don't saturate and might turn into on negative inputs.